app/styles/ember-arcgis-portal-components.scss

```diff
@@ -228,11 +227,10 @@ div.panel-body:has(.item-picker) {
       .shared-by-owner {
         font-size: 14px;
         display: inline-block;
-        margin: 0px auto;
       }
       .item-picker-current-item-preview-description {
         overflow: hidden;
-        height: 55px;
+        height: 70px;
```

Comment:
This one is probably going to be tricky because of that fadeout element on the bottom. But the main point of using flexbox is so that the layout is _flexible_ and we don't have to hard code height/width values like this.

Reply:
Could this be a percentage instead of `px` values?